web/packages/teleport/src/Roles/RoleEditor/YamlEditor.tsx

```diff
@@ -39,13 +42,25 @@ export const YamlEditor = ({
   yamlEditorModel,
   onChange,
   onSave,
+  onPreview,
   onCancel,
 }: YamlEditorProps) => {
   const isEditing = !!originalRole;
+  const [hasPreviewed, setHasPreviewed] = useState(!onPreview);
 
   const handleSave = () => onSave?.(yamlEditorModel.content);
 
+  const getHandlePreview = () => {
```

Comment:
Why is it called `getHandlePreview` and not `handlePreview`?

Reply:
was originally "getting" the function, but now that it isnt, i can rename. thank you!